dev/2017/spring/alumni-chapters-and-clubs/alumni-chapters-and-clubs.html

```diff
@@ -0,0 +1,66 @@
+<article class="story" id="knights-kingdom">
+  <div class="container-wide story-header-image" [background-image filename='alumni-map']>
+    <div class="container">
+      <div class="col-sm-6">
+        <h1>Knights&rsquo; Kingdom</h1>
+      </div>
+    </div>
+  </div>
+  <section id="intro">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+          <p>UCF alumni chapters and clubs are a great way to stay connected to everything you love about being a Knight. From prime networking events to fun tailgating parties, rewarding volunteer opportunities and much more, the experience of being an active member in your local group can bring benefits far beyond your degree.</p>
+          <p>&ldquo;With an alumni base of more than 263,000, it’s important to harness our scale and strengthen UCF&rsquo;s connection to our alumni through our chapters and clubs program,&rdquo; says Julie Stroh, senior associate vice president for advancement, alumni engagement and annual giving. &ldquo;Our regional groups are an integral part to helping us build a culture of service and philanthropy in support of our beloved university. Although we are in the very early stages of our international outreach, I can&rsquo;t wait to see what the future has in store as we broaden UCF’s global footprint.&rdquo;</p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section id="questions">
+    <div class="well well-ucf-gold">
+      <div class="container">
+        <div class="row">
+          <div class="col-sm-6 col-md-4 col-md-offset-2">
+            <h3 class="text-uppercase font-sans-serif h4">How can I get notified about events in my city?</h3>
+            <p>Be sure to update your information at <a href="https://ucfalumni.com/contactupdates">ucfalumni.com/contactupdates</a>. As long as we have your correct zipcode and email address, you’ll receive regular updates about social and networking events in your area.</p>
+          </div>
+          <div class="col-sm-6 col-md-4">
+            <h3 class="text-uppercase font-sans-serif h4">How can I get involved?</h3>
+            <p>Sign up to volunteer or nominate a friend to help you lead an alumni chapter or club at <a href="https://ucfalumni.com/nominationform">ucfalumni.com/nominationform</a>.</p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-sm-6 col-md-4 col-md-offset-2">
+            <h3 class="text-uppercase font-sans-serif h4">What's the difference between a chapter and a club?</h3>
+            <p>Chapters designate the largest, most active alumni groups, while clubs are for areas with fewer Knights.</p>
+          </div>
+          <div class="col-sm-6 col-md-4">
+            <h3 class="text-uppercase font-sans-serif h4">Have Additional Questions?</h3>
+            <p>Contact us at <a href="mailto:knightsaroundthenation@ucfalumni.com">knightsaroundthenation@ucfalumni.com.</a></p>
```

Comment:
The "knightsaroundthenation" email address causes a horizontal scrollbar between ~320-360px wide--consider reducing the font size at -xs, and/or setting text-overflow:ellipsis on the link.